crates/shared/src/price_estimation/competition/mod.rs

```diff
@@ -32,6 +32,8 @@ pub struct CompetitionEstimator<T> {
     stages: Vec<PriceEstimationStage<T>>,
     usable_results_for_early_return: NonZeroUsize,
     ranking: PriceRanking,
+    // TODO drop this flag and always prefer verified when feature is stable
```

Comment:
nit maybe create and reference issue to cleanup (this way once the issue is tackled we can simply search the codebase for the number and find all cleanup areas)

Reply:
Created https://github.com/cowprotocol/services/issues/2394 and removed the TODO comment because the issue is self explanatory.